experimental/traffic-portal/src/app/shared/shared.module.ts

```diff
@@ -81,7 +81,8 @@ import { CustomvalidityDirective } from "./validation/customvalidity.directive";
 		AppUIModule,
 		CommonModule,
 		RouterModule,
-		MatMenuModule
+		MatMenuModule,
+		NgOptimizedImage
```

Comment:
this is imported in the root app module as well; does it need to be imported in multiple places?